example/run_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Put your name, the source (to be counted) and the command used run you solution
+NAME=plusuncold
```

Comment:
this should be USER, no?

Reply:
I mean, the three lines are all specific examples. `NAME=USER` might actually be confusing.